.gitignore

```diff
@@ -15,6 +15,8 @@ project/plugins/project/
 .history
 .cache
 .lib/
+.idea/
```

Comment:
Can you delete these changes please? Its recommended to use a global `.gitignore` file for IDE specific files (see https://stackoverflow.com/questions/7335420/global-git-ignore)